x-pack/filebeat/module/aws/vpcflow/test/tcp-flag-sequence-skip-data.log-expected.json

```diff
@@ -0,0 +1,64 @@
+[
+    {
+        "@timestamp": "2019-08-26T19:48:53.000Z",
+        "aws.vpcflow.account_id": "123456789010",
+        "aws.vpcflow.action": "-",
```

Comment:
So it looks like we still have some "-" in aws, not sure if that is a problem.  Marius has been removing empty fields or empty place holder data with a painless script.  I did a quick modification and I think this will work (untested):


```
- script:
    lang: painless
    if: ctx?.aws != null
    source: |
      void handleMap(Map map) {
        for (def x : map.values()) {
          if (x instanceof Map) {
              handleMap(x);
          } else if (x instanceof List) {
              handleList(x);
          }
        }
        map.values().removeIf(v -> v instanceof String && v == "-");
      }
      void handleList(List list) {
        for (def x : list) {
            if (x instanceof Map) {
                handleMap(x);
            } else if (x instanceof List) {
                handleList(x);
            }
        }
      }
      handleMap(ctx.aws);
```

Reply:
👍 I will try to remove all "-"! Thanks!